api/src/main/java/org/pmiops/workbench/cdr/dao/CBCriteriaDao.java

```diff
@@ -440,6 +440,40 @@ List<DbSurveyVersion> findSurveyVersionByQuestionConceptIdAndAnswerConceptId(
       nativeQuery = true)
   List<DbCriteria> findVersionedSurveys();
 
+  @Query(
+      value =
+          "select distinct concept_id "
+              + "from cb_criteria c "
+              + "join ( "
+              + "      select cast(id as char) as id "
+              + "      from cb_criteria "
+              + "      where concept_id in (1740639) "
+              + "      and domain_id = 'SURVEY' "
+              + "      ) a on (c.path like CONCAT('%', a.id, '.%')) "
+              + "where domain_id = 'SURVEY' "
+              + "and type = 'PPI' "
+              + "and subtype = 'QUESTION' "
+              + "and concept_id in (:conceptIds)",
+      nativeQuery = true)
+  List<Long> findPFHHSurveyQuestionIds(@Param("conceptIds") List<Long> conceptIds);
+
+  @Query(
+      value =
+          "select distinct cast(value as signed) "
+              + "from cb_criteria c "
+              + "join ( "
+              + "      select cast(id as char) as id "
+              + "      from cb_criteria "
+              + "      where concept_id in (1740639) "
+              + "      and domain_id = 'SURVEY' "
+              + "      ) a on (c.path like CONCAT('%', a.id, '.%')) "
+              + "where domain_id = 'SURVEY' "
+              + "and type = 'PPI' "
+              + "and subtype = 'ANSWER' "
+              + "and concept_id in (:conceptIds)",
+      nativeQuery = true)
+  List<Long> findPFHHSurveyAnswerIds(@Param("conceptIds") List<Long> conceptIds);
```

Comment:
This is the lookup for answers in the PFHH survey(1740639)

Reply:
so the conceptIds are always question_concept_ids right? and this query is against MySQL and not BQ